frontend/src/component/project/Project/ProjectFeatureToggles/PaginatedProjectFeatureToggles.tsx

```diff
@@ -0,0 +1,666 @@
+import { createLocalStorage } from 'utils/createLocalStorage';
+import EnvironmentStrategyDialog from 'component/common/EnvironmentStrategiesDialog/EnvironmentStrategyDialog';
+import { FeatureStaleDialog } from 'component/common/FeatureStaleDialog/FeatureStaleDialog';
+import { FeatureArchiveDialog } from 'component/common/FeatureArchiveDialog/FeatureArchiveDialog';
+import { getColumnValues, includesFilter, useSearch } from 'hooks/useSearch';
+import { Search } from 'component/common/Search/Search';
+import { IFeatureToggleListItem } from 'interfaces/featureToggle';
+import { FavoriteIconHeader } from 'component/common/Table/FavoriteIconHeader/FavoriteIconHeader';
+import { FavoriteIconCell } from 'component/common/Table/cells/FavoriteIconCell/FavoriteIconCell';
+import {
+    ProjectEnvironmentType,
+    useEnvironmentsRef,
+} from './hooks/useEnvironmentsRef';
+import { ActionsCell } from './ActionsCell/ActionsCell';
+import { ColumnsMenu } from './ColumnsMenu/ColumnsMenu';
+import { useStyles } from './ProjectFeatureToggles.styles';
+import { usePinnedFavorites } from 'hooks/usePinnedFavorites';
+import { useFavoriteFeaturesApi } from 'hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi';
+import { FeatureTagCell } from 'component/common/Table/cells/FeatureTagCell/FeatureTagCell';
+import { useGlobalLocalStorage } from 'hooks/useGlobalLocalStorage';
+import FileDownload from '@mui/icons-material/FileDownload';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { ExportDialog } from 'component/feature/FeatureToggleList/ExportDialog';
+import { MemoizedRowSelectCell } from './RowSelectCell/RowSelectCell';
+import { BatchSelectionActionsBar } from 'component/common/BatchSelectionActionsBar/BatchSelectionActionsBar';
+import { ProjectFeaturesBatchActions } from './ProjectFeaturesBatchActions/ProjectFeaturesBatchActions';
+import { MemoizedFeatureEnvironmentSeenCell } from 'component/common/Table/cells/FeatureSeenCell/FeatureEnvironmentSeenCell';
+import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
+import { ListItemType } from './ProjectFeatureToggles.types';
+import { createFeatureToggleCell } from './FeatureToggleSwitch/createFeatureToggleCell';
+import { useFeatureToggleSwitch } from './FeatureToggleSwitch/useFeatureToggleSwitch';
+
+const StyledResponsiveButton = styled(ResponsiveButton)(() => ({
+    whiteSpace: 'nowrap',
+}));
+
+interface IPaginatedProjectFeatureTogglesProps {
+    features: IProject['features'];
+    environments: IProject['environments'];
+    loading: boolean;
+    onChange: () => void;
+    total?: number;
+    searchValue: string;
+    setSearchValue: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const staticColumns = ['Select', 'Actions', 'name', 'favorite'];
+
+const defaultSort: SortingRule<string> & {
+    columns?: string[];
+} = { id: 'createdAt' };
+
+export const PaginatedProjectFeatureToggles = ({
```

Comment:
This is same as ProjectFeatureToggles, but pulling out a lot of dependencies. Not using frontend search/filtering anymore, but features coming in from backend